src/NuGet.Clients/PackageManagement.UI/Converters/ProjectAndSolutionViewMinHeightConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI 
+{
+    class ProjectAndSolutionViewMinHeightConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var isSolution = (bool)value;
+            if (isSolution)
+            {
+                return 200.0;
```

Comment:
This number is device independent pixel so the size will be adjusted by WPF based on the current resolution.